docs/hsi.xml

```diff
@@ -128,6 +128,12 @@
         connected, additional software to be installed, or disabling any existing
         security layers to measure.
       </para>
+      <para>
+        The HSI specification is primarily designed for consumer laptop and
```

Comment:
Consumer?  I would think it's actually more valuable in business client.

Reply:
`designed for business and consumer laptop` perhaps?

Reply:
Maybe just "laptops"

Reply:
Agree. fixed.